libcxx/include/__ranges/drop_view.h

```diff
@@ -90,6 +90,10 @@ namespace ranges {
       requires (!(__simple_view<_View> &&
                   random_access_range<const _View> && sized_range<const _View>))
     {
+      if constexpr (random_access_range<const _View> && sized_range<const _View>) {
```

Comment:
This overload is non-const, so you should check the non-const _View

note that the requires clause uses const for a completely different reason: the spec tries to remove this non-const instantiation if non-const and const overload are exactly the same (we (sg9) always assume that  if the type of begin and begin const are the same , they are exactly the same). So the requires clause checks const to match the const overload. 

Here you are simply using the non-const base view so the check should be on non const View

Reply:
You're right. Thank you.